sdk/communication/azure-communication-phonenumbers/pom.xml

```diff
@@ -49,8 +49,8 @@
   <properties>
     <src.dir>src/main</src.dir>
     <test.dir>src/test</test.dir>
-    <jacoco.min.linecoverage>0.40</jacoco.min.linecoverage>
-    <jacoco.min.branchcoverage>0.50</jacoco.min.branchcoverage>
+    <jacoco.min.linecoverage>0.64</jacoco.min.linecoverage>
```

Comment:
Can we have same coverage numbers among all communication SDK? I see identity has different valueshttps://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/azure-communication-identity/pom.xml

Reply:
@sacheu The issue with Phone numbers is that we are not able to test the purchase and release APIs in live mode so that is why the coverage here is significantly lowered and I am not sure we can do anything about it ATM.